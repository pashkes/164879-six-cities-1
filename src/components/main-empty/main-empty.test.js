import React from "react";
import {shallow, configure} from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import MainEmpty from "./../main-empty/main-empty.jsx";

configure({adapter: new Adapter()});

it(`Empty main page renders correctly`, () => {
  const emptyMainPage = shallow(
      <MainEmpty selectedCity={`Kiev`}/>
  );
  expect(emptyMainPage).toMatchSnapshot();
});
