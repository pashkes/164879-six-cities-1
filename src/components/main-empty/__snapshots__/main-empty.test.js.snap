// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty main page renders correctly 1`] = `
<main
  className="page__main page__main--index page__main--index-empty"
>
  <h1
    className="visually-hidden"
  >
    Cities
  </h1>
  <div
    className="cities tabs"
  >
    <ConnectFunction
      cities={
        Array [
          "Paris",
          "Cologne",
          "Brussels",
          "Amsterdam",
          "Hamburg",
          "Dusseldorf",
          "Kiev",
        ]
      }
    />
  </div>
  <div
    className="cities__places-wrapper"
  >
    <div
      className="cities__places-container cities__places-container--empty container"
    >
      <section
        className="cities__no-places"
      >
        <div
          className="cities__status-wrapper tabs__content"
        >
          <b
            className="cities__status"
          >
            No places to stay available
          </b>
          <p
            className="cities__status-description"
          >
            We could not find any property availbale at the moment in 
            Kiev
          </p>
        </div>
      </section>
      <div
        className="cities__right-section"
      />
    </div>
  </div>
</main>
`;
